Content/Console/README.md

```diff
@@ -94,16 +94,51 @@ git push -u origin master
   - You can then set the `GITHUB_TOKEN` to upload release notes and artifacts to github
   - Otherwise it will fallback to username/password
 
-- Then update the `RELEASE_NOTES.md` with a new version, date, and release notes [ReleaseNotesHelper](https://fsharp.github.io/FAKE/apidocs/fake-releasenoteshelper.html)
+- Then update the `CHANGELOG.md` and add a new header with a version number and date just under the "Unreleased" header,
+  in [KeepAChangelog](https://keepachangelog.com/en/1.1.0/) format. Don't forget to make the version number
+  a link and update the link reference section at the bottom of the file, too. E.g., change this:
```

Comment:
Since we're generating these links now, I don't think this is necessary anymore.

Reply:
Agreed. I'll include that in the same commit as the change to the version number sanity checks.

Reply:
Done in dbc1a55.